core/src/main/java/org/apache/calcite/sql/SqlDeallocatePrepare.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql;
+
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.ImmutableNullableList;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Parse tree for {@code SqlDeallocatePrepare} call.
+ */
+public class SqlDeallocatePrepare extends SqlCall {
+  private static final SqlSpecialOperator OPERATOR =
+      new SqlSpecialOperator("DEALLOCATE PREPARE", SqlKind.DEALLOCATE_PREPARE);
+
+  public final SqlIdentifier statementName;
+
+  /**
+   * Creates a {@code SqlDeallocatePrepare}.
+   *
+   * @param pos  Parser position, must not be null
+   * @param statementName  Name of the statement to deallocate, must not be null
```

Comment:
nit: Line wrap

Reply:
It looks like this line is also just at 80 chars. Is it supposed to be < 80 or is <=80 fine?

Reply:
You're right - again! I just realized the helper utility I'm using to determine line lengths in these PRs had an off-by-one error. Should be fixed now.